src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/RunReportViewModel.kt

```diff
@@ -8,11 +8,21 @@ import org.vaccineimpact.orderlyweb.models.RunReportMetadata
 data class RunReportViewModel(
         @Serialise("runReportMetadataJson") val runReportMetadata: RunReportMetadata,
         @Serialise("gitBranchesJson") val gitBranches: List<String>,
-        val appViewModel: AppViewModel)
-    : AppViewModel by appViewModel
+        @Serialise("reportNameJson") val reportName: String?,
```

Comment:
This is serialised as a plain string rather than a JSON object, so you can just call it `reportName`

Reply:
I believe view-model require it to be serialised otherwise it doesn't work.